cmd/scollector/collectors/processes_windows.go

```diff
@@ -40,7 +40,7 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 	}
 
 	var svc_dst []Win32_Service
-	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	var svc_q = wmi.CreateQuery(&svc_dst, "")
```

Comment:
https://github.com/bosun-monitor/bosun/blob/master/util/Windows_WMI_Metadata.linq is a linqpad script that can do some simple code generation for us. Try running it from ny-rdp01.

Basically we use the github.com/stackexchange/wmi library to use reflection to create a WQL query based on a struct.